UpdateDependencies.ps1

```diff
@@ -32,8 +33,14 @@ function UpdateValidDependencyVersionsFile
 
     $DirPropsPath = "$PSScriptRoot\dir.props"
     
-    $DirPropsContent = Get-Content $DirPropsPath | % { 
-        $_ -replace "<CoreFxExpectedPrerelease>.*</CoreFxExpectedPrerelease>","<CoreFxExpectedPrerelease>$CoreFxLatestVersion</CoreFxExpectedPrerelease>"
+    $DirPropsContent = Get-Content $DirPropsPath | % {
+        $line = $_
+        $DirPropsVersionElements | % {
+            $line = $line -replace `
+                "<$_>.*</$_>", `
+                "<$_>$CoreFxLatestVersion</$_>"
```

Comment:
We should probably change this variable name to be LatestVersion to remove CoreFx.